crates/staged-sync/src/lib.rs

```diff
@@ -2,3 +2,7 @@ pub mod config;
 pub use config::Config;
 
 pub mod utils;
+
+#[cfg(any(test, feature = "test-utils"))]
+/// Common helpers for integration testing.
+pub mod test_utils;
```

Comment:
I wonder if staged-sync is the right crate here, I guess so we can use the pipeline and test certain parts of it against a GethInstance?

perhaps it would be appropriate to move to its own crate?

wdyt @onbjerg ?